.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,56 @@
+name: Bug report
+description: Create a report to help us improve
+title: '[issue summary] Please verify version before submitting new issue'
+labels:
+  - 'Type: Bug'
+  - not-reviewed
+  - auro-header
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report!
+  - type: input
+    id: version
+    attributes:
+      label: Please verify the version of auro-header you have installed
+  - type: markdown
+    attributes:
+      value: >
+        [![See it on
+        NPM!](https://img.shields.io/npm/v/@alaskaairux/auro-header?style=for-the-badge&color=orange)](https://www.npmjs.com/package/@alaskaairux/auro-header)
+  - type: textarea
+    id: details
+    attributes:
+      label: Please describe the bug
+      description: List out the steps to reproduce the behavior
+      placeholder: 1. Go to '...' 2. Click on '....' 3. Scroll down to '....' 4. See error
+  - type: dropdown
+    id: repro
+    attributes:
+      label: Reproducing the error
+      description: Are you able to reproduce this issue on the [Auro docsite](https://auro.alaskaair.com/)?
+      options:
+        - Yes I can
+        - No I can't
```

Comment:
Please reference this commit

https://github.com/AlaskaAirlines/auro-badge/commit/7da226006c43fdd0ad26c49abd84612d11b2b4aa